flow/project.py

```diff
@@ -174,8 +173,11 @@ def _isfile(*jobs):
 
     @classmethod
     def true(cls, key):
-        """True if the specified key is present in the job document(s) and
-        evaluates to True."""
+        """Evaluate job document key for True value.
```

Comment:
```suggestion
        """Evaluate job document key(s) for True value.
```

Reply:
This suggestion seems incorrect. The `get` method is called for a single key. The `all(...)` is over jobs, not keys.

Reply:
Yeah I agree that my proposed suggested wording is problematic. Perhaps "Check if key is True in all job documents."? The plural is important, but it also needs to qualify the correct object.

Reply:
I agree with what you're aiming for here. I suggest:
```suggestion
        """Evaluate if key is True in all job documents.
```
but the "all" may be unclear (project-wide vs. just this aggregate).

I like the statement below best, what do you think? If you agree, feel free to accept this suggestion and do the same for `false` below.
```suggestion
        """Evaluate if a document key is True for the job(s).
```

Reply:
Yes that seems like the best of the options we've considered.